Kickstarter-iOS/Views/Cells/PledgePaymentMethodCell.swift

```diff
@@ -0,0 +1,223 @@
+
+    _ = self.checkmarkImageView
+      |> checkmarkImageViewStyle
+
+    _ = self.lastFourLabel
+      |> lastFourLabelStyle
+
+    _ = self.expirationDateLabel
+      |> expirationDateLabelStyle
+  }
+
+  // MARK: - View model
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.lastFourLabel.rac.accessibilityLabel = self.viewModel.outputs.cardNumberAccessibilityLabel
+    self.lastFourLabel.rac.textColor = self.viewModel.outputs.lastFourLabelTextColor
+    self.expirationDateLabel.rac.text = self.viewModel.outputs.expirationDateText
+    self.lastFourLabel.rac.text = self.viewModel.outputs.cardNumberTextShortStyle
+    self.unavailableCardTypeLabel.rac.hidden = self.viewModel.outputs.unavailableCardLabelHidden
+    self.unavailableCardTypeLabel.rac.text = self.viewModel.outputs.unavailableCardText
+
+    self.viewModel.outputs.cardImageAlpha
+      .observeForUI()
+      .observeValues { [weak self] opacity in
+        self?.cardImageView.alpha = opacity
+      }
+
+    self.viewModel.outputs.checkmarkImageHidden
+      .observeForUI()
+      .observeValues { [weak self] hidden in
+        self?.checkmarkImageView.alpha = hidden ? 0 : 1
+      }
+
+    self.viewModel.outputs.cardImageName
+      .observeForUI()
+      .observeValues { [weak self] imageName in
+        _ = self?.cardImageView
+          ?|> \.image .~ Library.image(named: imageName)
+      }
+
+    self.viewModel.outputs.checkmarkImageName
+      .observeForUI()
+      .observeValues { [weak self] imageName in
+        _ = self?.checkmarkImageView
+          ?|> \.image .~ Library.image(named: imageName)
+      }
+
+    self.viewModel.outputs.selectionStyle
+      .observeForUI()
+      .observeValues { [weak self] style in
+        self?.selectionStyle = style
+      }
+  }
+
+  func configureWith(value: PledgePaymentMethodCellData) {
+    self.viewModel.inputs.configureWith(value: value)
+  }
+
+  // MARK: - Accessors
+
+  func setSelectedCard(_ card: GraphUserCreditCard.CreditCard) {
+    self.viewModel.inputs.setSelectedCard(card)
+  }
+}
+
+// MARK: - Styles
+
+private let adaptableStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> \.spacing .~ Styles.grid(2)
+}
+
+private let checkmarkImageViewStyle: ImageViewStyle = { imageView in
+  imageView
+    |> \.contentMode .~ .center
+}
+
+private let columnStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> \.axis .~ .vertical
+    |> \.spacing .~ Styles.grid(2)
+}
+
+private let expirationDateLabelStyle: LabelStyle = { label in
+  label
+    |> checkoutTitleLabelStyle
+    |> \.font .~ UIFont.ksr_caption2()
+    |> \.textColor .~ .ksr_dark_grey_500
+}
+
+private let labelsStackViewStyle: StackViewStyle = { stackView in
+  stackView
+    |> \.axis .~ .vertical
+    |> \.spacing .~ Styles.grid(1)
+}
+
+private let lastFourLabelStyle: LabelStyle = { label in
+  label
+    |> checkoutTitleLabelStyle
+    |> \.font .~ UIFont.ksr_caption1(size: 14).bolded
```

Comment:
Is it possible to use another font style that doesn't require a size override? Maybe `ksr_subhead`?

Reply:
I can try that.

Reply:
It increases kerning and size a little, will check in with @colleenmacd to see if we can switch it out.